### PR TITLE
feat: transaction fee ui

### PR DESCRIPTION
**Pull request by zhouhanseng**
Now only add gas price for creating ITO. And the usd estimation is not supported on testnet network like ropsten.

https://www.figma.com/file/gVkQ67y285b4FXVV1KPThN/Twitter?node-id=8475%3A1900

image
image
image

### CODE REVIEW

File: packages/maskbook/src/_locales/en/messages.json [hunk 1]
```diff
@@ -417,9 +418,10 @@
     "plugin_gas_now_dialog_min": "{{time}} Min",
     "plugin_gas_now_dialog_sec": "{{time}} Sec",
     "plugin_gas_now_dialog_gwei": "{{gasPrice}} Gwei",
-    "plugin_gas_now_dialog_usd": "≈ ${{usd}}",
+    "plugin_gas_now_dialog_usd": " ≈ ${{usd}}",
     "plugin_gas_now_dialog_zero_gas": "Invalid: Gas price should be greater than 0",
-    "plugin_gas_now_dialog_gas_price": "Gas price:",
+    "plugin_gas_now_dialog_gas_fee": "{{fee}} ETH",
```
Comment (guanbinrui) [inline packages/maskbook/src/_locales/en/messages.json:423]: Maybe other chains.

File: packages/maskbook/src/plugins/ITO/constants.ts [hunk 2]
```diff
@@ -150,3 +150,6 @@ export const QUALIFICATION_HAS_START_TIME_INTERFACE_ID = '0xdf29dfc4'
 export const QUALIFICATION_HAS_LUCKY_INTERFACE_ID = '0xadaa0f8a'
 
 export const MSG_DELIMITER = '2c1aca02'
+
+// for estimate gas
+export const FACK_SIGN_PASSWORD = '0x75466cc969717b172b14253aaeebdc958f2b5037a852c1337650ed4978242dd9'
```
Comment (guanbinrui) [inline packages/maskbook/src/plugins/ITO/constants.ts:155]:
```suggestion
export const FAKE_SIGN_PASSWORD = '0x75466cc969717b172b14253aaeebdc958f2b5037a852c1337650ed4978242dd9'
```

File: packages/maskbook/src/web3/UI/TxFeeEstimation.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import { GasNow, useGasPrice, useNetworkType, NetworkType, EthereumTokenType } from '@dimensiondev/web3-shared'
+import { currentGasNowSettings } from '../../plugins/Wallet/settings'
+import { useState, useCallback, useMemo } from 'react'
+
+const useStyles = makeStyles(() => {})
+
+interface TxFeeEstimationProps extends withClasses<'label' | 'button' | 'data' | 'gasEstimation'> {
+    gas?: number
+}
+const TUNE_ICON_DARK_MODE = new URL('../assets/tune_dark_mode.png', import.meta.url).toString()
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/UI/TxFeeEstimation.tsx:17]: Why not to use MUI icons?

Comment (zhouhanseng) [reply packages/maskbook/src/web3/UI/TxFeeEstimation.tsx:17]: MUI icons is not as same as our design on figma.

**Review by guanbinrui — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**